BUILDING.md

```diff
@@ -43,15 +43,15 @@ and can build the native library (UsdCs), the C# library (USD.NET) and the tests
  * gcc 7
  
  ##### OSX
- * xcode
+ * XCode
  
 #### Building USD 
 
-We typically use the standard build steps from the USD repo instructions. The exact command line we are using are:
+We typically use the standard build steps from the [USD repo instructions](https://github.com/PixarAnimationStudios/USD#getting-and-building-the-code). The exact command lines we are using are:
 * with python support: `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08-python36/usd-v20.08`
 * no python support:   `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-python --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08-python36/usd-v20.08_no_python`
 
-**Unity developers**: USD binaries are built with each new release and pushed on Stevedore. See next section to download them automatically when building the bindings.
+**Unity developers**: USD binaries are built with each new release and pushed on Stevedore. See next section to download them automatically when building the bindings instead of building them.
```

Comment:
Might reorder this sentence to:
Instead of building them, see the next section to download them automatically when building the bindings.